Create ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md (#2481)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1020-28-12-fixed-issues-in-the-asset-properties-widget-on-device-info-page.md

---
date: ""

> title: Asset properties widget on device info page fixed

Reviewer: ```suggestion
title: Asset properties widget in device details correctly displays all relevant properties

```

> title: Fixed issues in the asset properties widget on device info page.

Author: The Claude AI suggested to use the following title:
```suggestion
title: Asset properties widget on device info page fixed
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3868
version: 1020.28.12
---

> Fixed issues in the asset properties widget on device info page.

Reviewer 2: ```suggestion
The asset properties widget on the device info page was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the asset properties widget now correctly displays all relevant properties for a device asset. The updated asset properties widget in the device info page now contains 5 default properties - ID, name, type, last updated and creation time.
```

> Fixed issues in the asset properties widget on device info page.

Author: The Claude AI suggested to use the following body:
```suggestion
The asset properties widget on the device info page was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the asset properties widget now correctly displays all relevant properties for a device asset. Users relying on the asset properties widget for device information can now be assured the data shown is accurate and complete.
```

> The asset properties widget on the device info page was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the asset properties widget now correctly displays all relevant properties for a device asset. The updated asset properties widget in the device info page now contains 5 default properties - ID, name, type, last updated and creation time.

Reviewer: ```suggestion
The "Asset properties" widget on the **Info** tab in the device details was not working as expected and showed incorrect or missing information in some cases. This issue has been resolved and the "Asset properties" widget now correctly displays all relevant properties for a device asset. It contains 5 default properties - ID, name, type, last updated and creation time.
```
